src/main/java/uk/gov/hmcts/reform/hmc/helper/hmi/HmiSubmitHearingRequestMapper.java

```diff
@@ -36,4 +46,20 @@ public HmiSubmitHearingRequest mapRequest(Long hearingId, HearingRequest hearing
             .build();
     }
 
+    /**
+     * map Delete hearing Request to HMI Submit Delete Hearing Request.
+     * @param hearingRequest delete hearing request
+     * @return hmiSubmitDeleteHearingRequest HMI Submit Delete Hearing request object
+     */
+    public HmiSubmitDeleteHearingRequest mapRequest(DeleteHearingRequest hearingRequest) {
```

Comment:
```suggestion
    public HmiDeleteHearingRequest mapRequest(DeleteHearingRequest hearingRequest) {
```
Can we also move this to a separate class please as this class is for mapping Submit Hearing requests. `HmiDeleteHearingRequestMapper` would be more appropriate for this method to sit in

Reply:
Done